dora/underfs/gcs/src/main/java/alluxio/underfs/gcs/v2/GCSV2UnderFileSystem.java

```diff
@@ -137,6 +139,32 @@ public PositionReader openPositionRead(String path, long fileLength) {
   @Override
   public void setOwner(String path, String user, String group) {}
 
+  @Override
+  public void setObjectTagging(String path, String name, String value) throws IOException {
+    Map<String, String> newMetadata = new HashMap<>();
+    newMetadata.put(name, value);
```

Comment:
this actually creates a Map of 16 instances. What about just `ImmutableMap.of(name, value)`?